### ISSUE TITLE
Sometimes Enter button don't initiate send message but adds new line always

### ISSUE THREAD

**Issue by MurzNN** [vector-im/riot-web#4961]
Pressing Enter button sends message in most cases, but after update to Riot 0.12.2 (using Electron on Windows 8.1)- sometimes I can't send message - each Enter adds new line, and triple & more Enter presses don't send message, so there are no way to send message, because separate Send button is missing too. 

Enable-disable markdown, turn on-off RTE panel don't help.

Helps only switch to another room, and switch back - after this Enter works normally.

Seems that new Riot RTE goes in some strange mode (maybe via some hotkey?) that accepts Enter as symbol always, and if I switch to another room and back - it re-initialized and works well.

**Comment by MurzNN** [comment]
Here https://github.com/vector-im/riot-web/issues/4931 is similar issue about Enter, but `Enter key 3 times` don't solve this issue.

**Comment by MurzNN** [comment]
This happens for me on https://riot.im/develop/ now when I click on username in chat log for mention, but this name is not added to textarea (textarea keeps empty), after this I try to type manually message text, and enter key adds lines instead of sending message. Switch to other room and back - solves the problem. So seems the bug is in `mention by click` script.

**Comment by lukebarnard1** [comment]
@MurzNN do you remember which name? Or is it reproducible with any name?

**Comment by MurzNN** [comment]
This is reproducible not each time, but some times randomly. I try to find way to reproduce now, but can not. I continue trying and report if find some dependencies.

**Comment by MurzNN** [comment]
I reproduce this error after random clicks on room, here is console error log:
```
Unhandled rejection getHostNode@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22822:6
getHostNode@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244283:13
getHostNode@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22822:13
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:243843:31
_reconcilerUpdateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248056:12
_updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248160:27
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248147:8
_updateDOMChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245748:8
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245562:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245524:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:243823:10
_reconcilerUpdateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248056:12
_updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248160:27
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248147:8
_updateDOMChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245748:8
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245562:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245524:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:243823:10
_reconcilerUpdateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248056:12
_updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248160:27
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248147:8
_updateDOMChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245748:8
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245562:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245524:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
_updateRenderedComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244653:8
_performComponentUpdate@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244623:6
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244544:8
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244446:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:243823:10
_reconcilerUpdateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248056:12
_updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248160:27
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248147:8
_updateDOMChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245748:8
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245562:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245524:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:243823:10
_reconcilerUpdateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248056:12
_updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248160:27
updateChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:248147:8
_updateDOMChildren@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245748:8
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245562:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:245524:6
receiveComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22884:6
_updateRenderedComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244653:8
_performComponentUpdate@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244623:6
updateComponent@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244544:8
performUpdateIfNecessary@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:244460:8
performUpdateIfNecessary@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:22916:6
runBatchedUpdates@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:14910:6
perform@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:43487:14
perform@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:43487:14
perform@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:14849:13
flushBatchedUpdates@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:14932:8
closeAll@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:43553:12
perform@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:43500:12
batchedUpdates@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:247378:15
enqueueUpdate@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:14960:6
enqueueUpdate@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:61188:4
enqueueSetState@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:61382:6
ReactComponent.prototype.setState@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:100564:4
setState@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:136161:15
MessageComposerInput/_this.setDisplayedCompletion</_ref4<@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:135909:18
tryCatcher@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:9464:17
module.exports/Promise.method/<@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:145863:22
MessageComposerInput/_this.setDisplayedCompletion</<@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:135917:25
MessageComposerInput/_this.onAction@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:135472:26
_invokeCallback@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:163057:6
dispatch@https://riot.im/develop/bundles/e12e836f8791d5326e90/bundle.js:163033:10
```

**Comment by ara4n** [comment]
fixed by matrix-org/matrix-react-sdk#1890
